specviz/plugins/statistics/statistics_widget.py

```diff
@@ -168,6 +172,9 @@ def _init_ui(self):
                         (doc.documentMargin() + widget.frameWidth()) * 2 +
                         margins.top() + margins.bottom())
             widget.setFixedHeight(n_height)
+        self.comboBox.addItems(["Generic", "Continuum Subtracted", "Continuum Normalized"])
```

Comment:
Just a suggestion: maybe "Basic" instead of "Generic" -- not sure what "generic" statistics are. 😝

Reply:
Changed in `b64d35d`